tests/gordo_components/data_provider/test_data_provider_iroc.py

```diff
@@ -16,6 +16,23 @@
 HAPPY_FROM_TS = isoparse("2018-05-02T01:56:00+00:00")
 HAPPY_TO_TS = isoparse("2018-05-03T01:56:00+00:00")
 
+IROC_MANY_ASSETS_TAG_LIST = [
```

Comment:
Okey! So something more came to my attention here. The testing you do here, sends in tags with asset=None. That will always fail in your code (as in never provide a path). If you look in the test_ncs_reader.py you see that we send in assets explitly, or use the "normalize_sensor_tags" in the sensor_tag.py file. This must be done here too, to make sense.